### ISSUE TITLE
Travis is timing out

### ISSUE THREAD

**Issue by jadephilipoom** [mit-plv/fiat-crypto#438]
Seems like it's actually timing out on the step where it updates Travis's own cache with new .vo files. Recent builds have been alternately failing on the first and third build stages.

We can fix this by any of the following:
- splitting out builds more
- switching to GitLab
- deprecating the old pipeline

**Comment by jadephilipoom** [comment]
Note: upon further investigation the cause of this is probably that some of the files in `src/Experiments/NewPipeline/` produce huge `.vo` files. See the following list of the top several `.vo` files sorted by bytes (some stuff in Specific might be larger, I don't have everything compiled):

```
29000398 src/Experiments/NewPipeline/Rewriter.vo
19020757 src/Experiments/NewPipeline/GENERATEDIdentifiersWithoutTypes.vo
8044911 src/Experiments/NewPipeline/RewriterRulesGood.vo
5394138 src/Experiments/NewPipeline/Toplevel1.vo
1945225 src/Experiments/NewPipeline/LanguageInversion.vo
1602091 src/Experiments/NewPipeline/AbstractInterpretationWf.vo
1445258 src/Experiments/NewPipeline/Arithmetic.vo
1248042 src/Experiments/NewPipeline/GENERATEDIdentifiersWithoutTypesProofs.vo
1161034 src/Experiments/NewPipeline/RewriterWf2.vo
869499 src/Experiments/NewPipeline/LanguageWf.vo
839034 src/Experiments/NewPipeline/CStringification.vo
805678 src/Experiments/NewPipeline/UnderLetsProofs.vo
687300 src/Util/ListUtil.vo
613222 src/Experiments/NewPipeline/AbstractInterpretationZRangeProofs.vo
601536 src/Experiments/NewPipeline/Language.vo
522513 src/Experiments/NewPipeline/AbstractInterpretationProofs.vo
518892 src/Util/FixedWordSizesEquality.vo
401220 src/Util/FsatzAutoLemmas.vo
391054 src/Experiments/NewPipeline/RewriterWf1.vo
386939 src/Util/WordUtil.vo
357261 src/Util/Tuple.vo
```

**Comment by JasonGross** [comment]
Should be fixed, for now, by #454
